### ISSUE TITLE
flow-line paths never get .active class / stuck at opacity:0 despite nodes correctly showing active state (v0.3.30)

### ISSUE THREAD

**Issue by NolanMike5150** [stexecute/tesla-style-energy-flow#36]
**Card version:** v0.3.30 (installed via HACS)

**Summary:**
The animated flow lines never render/animate, even when power values are well above the configured thresholds. This isn't a data issue — entities are reporting valid, live numbers — and it isn't the grid→home dash-stutter fixed in #27/v0.3.29. In my case, no flow line animates at all, for any path.

**Observed via browser devtools (querying the card's shadow root directly):**
- `circle.flow-node-bg` elements (6 total): 2 of them correctly receive the `active` class and brighter fill (`rgba(255,255,255,0.72)` vs `rgba(255,255,255,0.08)` for inactive), matching real battery discharge (~900W) and home load (~900W) — well above my `battery_min_w: 50` threshold.
- `path.flow-line` elements (9 total): **none of them ever receive an `active` class.** All 9 sit at `opacity: 0` regardless of the node states above. `animationName` is `none` on all of them.
- No particle/dot elements (searched for classes matching `particle|dot|flow-dot|energy-dot`) exist anywhere in the shadow root — 0 found.

**Config (relevant excerpt):**
```yaml
type: custom:tesla-style-energy-flow
thresholds:
  solar_min_w: 50
  grid_min_w: 50
  battery_min_w: 50
smoothing_seconds: 0
entities:
  solar_power: sensor.my_home_solar_power_3
  grid_power: sensor.my_home_grid_power_2
  battery_power: sensor.my_home_battery_power_3
  load_power: sensor.my_home_load_power_3
  battery_level: sensor.my_home_charge
  ev_power: sensor.gargar_mobile_charger_power_2
  ev_battery: sensor.gargar_mobile_battery_level
  ev_charge_switch: switch.gargar_mobile_charge
```

**Steps to reproduce:**
1. Have a battery discharging at a real wattage above `battery_min_w` (confirmed via Developer Tools → States that the entity is a real number, not `unknown`).
2. Load the card; observe the wattage labels update correctly (confirming reactive data flow), and the "HOME"/"BATTERY" node dots become visibly brighter (`active` class applied).
3. Observe: no line/particle animation ever appears connecting battery → home, even though both endpoint nodes show `active`.
4. Hard refresh (not just reactive update) does not resolve it.

**Note:** I initially suspected this was caused by an EV entity being `unknown` (a Tesla vehicle integration had gone stale for ~24h), since `ev_power` was `unknown` at the time. After fixing that (reloading the Tesla Fleet integration so `ev_power` returned to a real `0` value), the flow-line issue persisted identically — so the EV entity state is not the trigger.

Happy to provide a screen recording or run further devtools queries against the shadow DOM if useful — this seemed like a rendering/state-sync bug between the node-active logic and the line-active logic specifically, similar in spirit to the dash-cycle mismatch fixed in #27, but affecting whether the `active` class is applied at all rather than the animation's timing.

<img width="417" height="402" alt="Image" src="https://github.com/user-attachments/assets/c83e4f7b-bbaf-4094-99cf-b90ba19a8d40" />
